### ISSUE TITLE
img missing alt text

### ISSUE THREAD

**Issue by discospiff** [wjtreutel/IS7012#9]
Use alt text to give an accurate, yet concise description of what the image looks like, for the benefit of users who cannot see the image.

https://github.com/wjtreutel/IS7012/blob/735a9a71782051b372202b07d129c05831559c3b/Assignment 2/RecruitCatTreutewm/Pages/Index.cshtml#L61
